### PR TITLE
Show UI on text selection

### PR DESCRIPTION
**Pull request by ellatrix**
See #712.

### CODE REVIEW

**Comment by jasmussen** [comment]
I love how this works. 👍 👍  from me!

**Review by youknowriad — APPROVED** [review]
Good Job 🚢

File: blocks/editable/index.js [hunk 1]
```diff
@@ -131,10 +131,18 @@ export default class Editable extends wp.element.Component {
 		}
 
 		const content = this.getContent();
+		const collapsed = this.editor.selection.isCollapsed();
 
 		this.setState( {
 			empty: ! content || ! content.length,
 		} );
+
+		if ( this.props.focus.collapsed !== collapsed ) {
```
Comment (aduth) [inline blocks/editable/index.js]: Could we expect `focus` to ever be `undefined` at this point?

Comment (iseulde) [inline blocks/editable/index.js]: If somehow the editor is active and the block is not selected, then yes. Maybe `getBlockFocus` should return an empty object in that case? Or not? Cc @youknowriad.

Comment (youknowriad) [inline blocks/editable/index.js]: It think this is possible when the editor become active (before the rerender), it's worth checking IMO. good catch @aduth

**Pull request merged**